packages/nitro-protocol/test/contracts/TrivalApp/validTransition.test.ts

```diff
@@ -0,0 +1,33 @@
+// @ts-ignore
+import {expectRevert} from '@statechannels/devtools';
+import {Contract} from 'ethers';
+import {HashZero} from 'ethers/constants';
+import TrivialAppArtifact from '../../../build/contracts/TrivialApp.json';
+import {VariablePart} from '../../../src/contract/state.js';
+import {getTestProvider, setupContracts} from '../../test-helpers';
+
+const provider = getTestProvider();
+let trivialApp: Contract;
+
+beforeAll(async () => {
+  trivialApp = await setupContracts(provider, TrivialAppArtifact);
+});
+
+describe('validTransition', () => {
+  it.each`
+    isValid  | description
+    ${true}  | ${'IsValid True = True'}
+    ${false} | ${'IsValid False = True'}
```

Comment:
These tables are designed to be used for different test cases, for example each row might describe a different pair of `to` and `from` states to feed the `validTransition` function. As it is, only your *expectations* change (and even then not really) from row to row. 

The table format is really great but this test might not be the place where it really shines :-)